packages/core/admin/admin/src/content-manager/components/RepeatableComponent/AccordionGroupCustom/index.js

```diff
@@ -8,7 +8,6 @@ import { Flex } from '@strapi/design-system/Flex';
 import { KeyboardNavigable } from '@strapi/design-system/KeyboardNavigable';
 
 const AccordionFooter = styled(Box)`
-  overflow: hidden;
```

Comment:
`overflow: hidden;` rule caused `:focus-visible:after` - our designed focus outline - is cut off. As far as we can see, this change doesn't break current usages.